File: app/src/main/java/com/datarockets/mnchkn/fragments/SettingsFragment.java
package com.datarockets.mnchkn.fragments;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import com.datarockets.mnchkn.R;

public class SettingsFragment extends PreferenceFragment {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.preferences);
    }
}

File: app/src/main/kotlin/com/datarockets/mnchkn/ui/settings/SettingsFragment.kt
package com.datarockets.mnchkn.ui.settings

import android.os.Bundle
import android.preference.PreferenceFragment

import com.datarockets.mnchkn.R

class SettingsFragment : PreferenceFragment() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.preferences)
    }
}
